utils/helpers.js

```diff
@@ -267,6 +269,17 @@ const unPauseAndPoll = async(ERC20Pausable) => {
 
     return false // paused
 }
+
+const pollForResponseCode = async(contract) => {
+  for (let numberOfTries = 0; process.env.MAX_RETRY <= process.env.MAX_RETRY; numberOfTries++) {
+    const event = contract.events.filter(
+      (e) => e.event === Constants.Events.ResponseCode
+    )[0].args[0]
+    return parseInt(event._hex);
```

Comment:
Q: How does this work?
Won't the function return the on the first iteration every time?
There's no condition to decide if to return or not so i'm not sure how the for loop is providing value.
Did you mean to check for event characteristics?

Reply:
Push up the latest.  I am not sure what happened, but it was supposed to have a for loop, and it did on my box.  I may have missed staging the latest version.